roles/common/tasks/debian.yml

```diff
@@ -25,8 +25,11 @@
 
 - name: Add Confluent Apt Key
   apt_key:
-    url: "{{confluent_common_repository_debian_key_url}}"
+    url: "{{item}}"
     state: present
+  loop:
+    - "{{confluent_common_repository_debian_key_url}}"
+    - "{{confluent_control_center_next_gen_repository_debian_key_url}}"
```

Comment:
what would happen when confluent_control_center_next_gen_repository_debian_key_url is not defined? i.e. in case of non c3++ setup?